lib/node-utils.ts

```diff
@@ -253,3 +253,22 @@ export function getImportModuleName(
     return node.arguments[0].value;
   }
 }
+
+export function getSpecifierFromImport(
+  node: ImportModuleNode,
+  specifierName: string
+) {
+  if (isImportDeclaration(node)) {
+    const namedExport = node.specifiers.find(
+      (node) => isImportSpecifier(node) && node.imported.name === specifierName
+    );
+    // it is "import { foo } from 'baz'""
+    if (namedExport) {
+      return namedExport;
+    }
+    // it could be "import * as rtl from 'baz'"
+    return node.specifiers.find((n) => isImportNamespaceSpecifier(n));
+  } else {
+    // TODO make it work for require
+  }
+}
```

Comment:
if we move this function into the detect-import-testing-library utils, instead of passing `node` I could get it directly from the helper. Not sure which option is best.

I still have to make it work for both `require` scenarios mentioned

Reply:
I think it's a good idea to have this within detection module. I actually [mentioned this helper](https://github.com/testing-library/eslint-plugin-testing-library/issues/198#issuecomment-669927224) when analyzing how to organize the helpers for detection module. There are difference scenarios as you have already figured in your draft, but there are more related to other Testing Library utils, specially for `render`.

Anyway, I think we should include this one within detection already, covering all different imports. Then we can improve it when necessary when refactoring other rules. I'd name it something like `findTestingLibraryUtilNode`

Reply:
I moved it into the detection helper, and added the scenario for `require`